BuildingSync.xsd

```diff
@@ -13380,6 +13409,32 @@
                 <xs:documentation>Version number of building energy modeling software</xs:documentation>
               </xs:annotation>
             </xs:element>
+            <xs:element name="WeatherDataType" minOccurs="0">
+              <xs:annotation>
+                <xs:documentation>Type of weather data used for the simulation</xs:documentation>
+              </xs:annotation>
+              <xs:simpleType>
+                <xs:restriction base="xs:string">
+                  <xs:enumeration value="Typical Meteorological Year"/>
+                  <xs:enumeration value="Weather Station"/>
```

Comment:
I think this is going to need to have a few more values. I'd suggest that the minimum to add are "Other" and "Unknown", but with the various iterations of TMY and the international equivalents, it might be advisable to consider deleting "Meteorological" from the value so that it would cover all of those at once or adding TMY, TMY2, TMYX, IWEC, etc.

Reply:
@jasondegraw ... do we just use these items? 

https://bedes.lbl.gov/bedes-online/weather-data-type

Reply:
@nllong BEDES to the rescue again. Yes, that'd be the list to use.

Reply:
@kflemin -- can you update this PR with the list that we should have followed the first time...

Reply:
grumblecakes!  Yes, will do.  I also need to add a "PeakResourceUnits" to ResourceUses for consistency.

Reply:
ok done!

Reply:
@nllong Do we need to add "Other" and/or "Unknown"?